src/commands/init.js

```diff
@@ -80,6 +83,96 @@ async function handleDatabaseConfiguration() {
   return inquirer.prompt(promptContent);
 }
 
+function validateEndpoint(input) {
+  if (!VALIDATION_REGEX_URL.test(input)) {
+    return 'Application input must be a valid url.';
+  }
+  if (!VALIDATION_REGEX_HTTPS.test(input)) {
+    return 'HTTPS protocol is mandatory, except for localhost and 127.0.0.1.';
+  }
+  return true;
+}
+
+function getApplicationPortFromCompleteEndpoint(endpoint) {
+  if (endpoint.startsWith('http://localhost:') || endpoint.startsWith('http://127.0.0.1:')) {
+    return endpoint.substring(17);
+  }
+  return null;
+}
+
+async function getContentToAddInDotenvFile(config) {
+  const authSecret = await generateKey();
+  let contentToAddInDotenvFile = '';
+
+  if (config.applicationPort) {
+    contentToAddInDotenvFile += `APPLICATION_PORT=${config.applicationPort}\n`;
+  }
+  if (config.databaseUrl) {
+    contentToAddInDotenvFile += `DATABASE_URL=${config.databaseUrl}\n`;
+  }
+  if (config.databaseSchema) {
+    contentToAddInDotenvFile += `DATABASE_SCHEMA=${config.databaseSchema}\n`;
+  }
+  if (config.databaseSSL !== undefined) {
+    contentToAddInDotenvFile += `DATABASE_SSL=${config.databaseSSL}\n`;
+  }
+  contentToAddInDotenvFile += `FOREST_AUTH_SECRET=${authSecret}\n`;
+  contentToAddInDotenvFile += `FOREST_ENV_SECRET=${config.forestEnvSecret}`;
+  return contentToAddInDotenvFile;
```

Comment:
[Performance] Using `+=` to build string is a really bad practice in many coding languages (as it has to reallocate memory space for each addition) thus using `concat` should be preferred. However I'm not entirely sure that this applies to js

Reply:
After a quick search, it seems indeed that that performance issue doesn't happen in js. Let's stick with the `+=` then.

Reply:
I actually hesitate between `+=` and `concat`.
I choose `+=` because it looked cleaner to me.
Thanks for the research on the subject!